taichi/codegen/cuda/codegen_cuda.cpp

```diff
@@ -738,11 +738,14 @@ FunctionType CodeGenCUDA::codegen() {
   return CodeGenLLVMCUDA(kernel, ir).gen();
 }
 
+#ifdef TI_WITH_LLVM
+
 FunctionType CUDAModuleToFunctionConverter::convert(
     const std::string &kernel_name,
     const std::vector<LlvmLaunchArgInfo> &args,
-    std::unique_ptr<llvm::Module> mod,
-    std::vector<OffloadedTask> &&tasks) const {
+    std::vector<LLVMCompiledData> &&data) const {
+  auto &mod = data[0].module;
```

Comment:
Should be a for loop? 

```
for(auto& d : data) {
  auto& mod = d.module;
  ...
}
```

Reply:
This one is fake for now. I haven't added the support for multiple modules for CUDA. I'll add it later.